packages/components/src/color-picker/index.js

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
Can we add a corresponding README file with a simple usage example so it could be exposed in both Gutenberg Handbook and Calypso DevDocs?
A similar example: https://github.com/WordPress/gutenberg/blob/master/packages/components/src/color-palette/README.md.